class/defaults.yml

```diff
@@ -322,3 +322,12 @@ parameters:
       instance: null
 
     secrets: {}
+
+    cortex_tenant_ns_label:
+      enabled: false
+      requests:
+        cpu: '500m'
+        memory: '256Mi'
+      limits:
+        cpu: '5'
```

Comment:
Do we have any indication that this process will need up to 5 CPU cores? What's the expected average usage? In my understanding, the current CPU request and limit are too far apart, and if the process actually uses a significant fraction of its CPU limits, whatever scheduling decision K8s made won't make sense since the process might be scheduled on a node which only has ~half a CPU core available.

Reply:
I honestly have no idea. It currently uses about 0.5 cores on APPUiO Lab (which is more than I expected). This obviously depends on how many metrics Prometheus tries to send through the proxy, but we don't even know at this point which metrics we want to send to Mimir.